kinto/core/cache/memcached.py

```diff
@@ -86,7 +86,7 @@ def expire(self, key, ttl):
         if ttl == 0:
             self.delete(key)
         else:
-            # We can't use touch here because we need to update the TTL value in the record.
+            # We can't use touch here because we need to update the TTL value in the object.
```

Comment:
This isn't really a Kinto record but a Memcached record. I could kind of go either way on this.